### DISCUSSION TITLE
No htmlLink available in gcsa.event.Event

### DISCUSSION THREAD

**Discussion by donaldm314** [kuzmoyev/google-calendar-simple-api#193] [General]
Hi folks, 

I sought to use the htmlLink in an event I've read from a calendar, as returned by get_events(). However gcsa.event.Event has no htmlLink attribute.

After looking at the code here, I see htmlLink is not supported, and I don't see anyone else interested in this attribute. 

Is there another way to access htmlLink? Should I submit and issue and then work on a PR?

**Comment by kuzmoyev** [comment]
Hey @donaldm314.

You can find it in `other` field of the `Event`:

```
event.other['htmlLink']
```

Can you describe your usecase? Do you think it's useful and should be added as a proper field in the `Event`? If so, you can open an issue for that.

**Reply by donaldm314** [reply]
Ah, that works perfectly, thank you!

I am extracting events from a calendar into a spreadsheet, and I want to click on the htmlLink and see the original event in Google Calendar.

Since `event.other['htmlLink']` already provides the solution I seek, I don't think an issue is warranted.
